projects/swimlane/ngx-cron/src/lib/ngx-cron.service.ts

```diff
@@ -122,7 +168,8 @@ export class NgxCronService {
     }
   }
 
-  getCronData(cron: string, period: Period): ICronData {
+  getCronData(cron: string, period: Period, lang, configOverrides): ICronData {
```

Comment:
suggestion: we can type `configOverrides` with `typeof NgxCronService.CRON_VALIDATE_CONFIG_OVERRIDES`

Reply:
Done